keel-web/src/main/kotlin/com/netflix/spinnaker/config/DefaultConfiguration.kt

```diff
@@ -33,10 +35,39 @@ import org.springframework.context.annotation.Primary
 import org.springframework.core.Ordered.HIGHEST_PRECEDENCE
 import org.springframework.scheduling.TaskScheduler
 import org.springframework.scheduling.concurrent.ThreadPoolTaskScheduler
+import org.springframework.web.servlet.config.annotation.InterceptorRegistry
+import org.springframework.web.servlet.config.annotation.WebMvcConfigurer
+
+private const val IPC_SERVER_METRIC = "controller.invocations"
 
 @EnableFiatAutoConfig
 @Configuration
-class DefaultConfiguration {
+class DefaultConfiguration(
+  val spectatorRegistry: Registry
+) : WebMvcConfigurer {
+
+  /**
+   * Enable controller metrics
+   */
+  override fun addInterceptors(interceptorRegistry: InterceptorRegistry) {
+
+    /**
+     * Path variables that will be added as tags to the metrics
+     *
+     * Lorin chose these particular variables arbitrary as potentially useful. Once we start consuming
+     * these metrics, we should revisit whether to change these based on usefulness & cardinality
+     */
+    val pathVarsToTag = listOf("name", "application")
+
+    val queryParamsToTag = null
+
+    // exclude list copied from fiat: https://github.com/spinnaker/fiat/blob/0d58386152ad78234b2554f5efdf12d26b77d57c/fiat-web/src/main/java/com/netflix/spinnaker/fiat/config/FiatConfig.java#L50
+    val exclude = listOf("BasicErrorController")
```

Comment:
Is that controller auto-magically added by `kork`, or an actual controller in `fiat`? If the latter, then we should probably remove this for `keel`.

Reply:
I _think_ this is auto-magically added by the Spring Boot framework for serving error pages.